jools.html

```diff
@@ -0,0 +1,23 @@
+<DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../main.css" rel="stylesheet">
+    <title>Hello World</title>
+</head>
+<body>
+    <h1>Jools</h1>
+
+    <img src="file:///Users/jools/Desktop/Dog/211866867_173194114831481_9170958334940495567_n.jpg"
```

Comment:
Unfortunately, this link will not show up because it's local to your computer. 
Here's a few choices you can take: 
1. Search online for a picture and then right click and Copy Image Address and paste it
2. Use the default link I provided in the your-name.html file

Totally up to you which route you want to go

Reply:
Yes, I just realized that just after I sent in the changes and then found a link to actually connect.

Hope this change works.